### ISSUE TITLE
[FEATURE] method for actual/screen visibility?

### ISSUE THREAD

**Issue by MSP-Greg** [SketchUp/api-issue-tracker#989]
In the past, determining an Entity's actual or screen visibility involved more than one check.

With the addition of `LayerFolder`, it has gotten more complex due to the hierarchy of Folders.

Could a method be added to `Drawingelement` (or even `Entity`) that would return the actual/screen visibility?

Something like `actual_visibility?` or `screen_visible?` ?

See https://forums.sketchup.com/t/how-to-get-all-visible-entities/279529

**Comment by thomthom** [comment]
https://ruby.sketchup.com/Sketchup/Model.html#drawing_element_visible%3F-instance_method

We added  `model.drawing_element_visible?(instance_path)` in SU2020.

You need an instance path to determine if a drawing element is visible because it can exist in multiple instances in a model that might have different visibility.

Unless that method is missing something I'm closing this as "already implemented".

**Comment by MSP-Greg** [comment]
@thomthom

Good point.  I haven't written that much code related to visibility, and I think I normally start at top level, then drill in.  As you've mentioned, Components and Groups affect one's code if one uses actual 'low-level' items.

It would be helpful if something similar to `model.drawing_element_visible?(instance_path)` existed as a method of `Sketchup::InstancePath`
